build/repo.targets

```diff
@@ -97,7 +97,9 @@
     <Exec
       Command="$(_CloneArguments) --branch $(CloneBranch)"
       Condition="'$(GateBranchExitCode)'!='0'"
-      WorkingDirectory="$(_CloneRepositoryRoot)" />
+      WorkingDirectory="$(_CloneRepositoryRoot)"
+      EnvironmentVariables="GIT_TERMINAL_PROMPT=0"
+      Timeout="180000" />
```

Comment:
Yes. The first's to prevent git for asking for a password if the repository is private. This specifically works for Git v2.3 or higher (https://github.com/blog/1957-git-2-3-has-been-released#user-content-the-credential-subsystem-is-now-friendlier-to-scripting). The latter is to force a 3 minute timeout in case we're running git < 2.3 (like I was on my local box) and are stuck with a prompt that won't be responded to.